src/ansys/geometry/core/designer/component.py

```diff
@@ -174,15 +175,24 @@ def __init__(
         if preexisting_id:
             self._name = name
             self._id = preexisting_id
+            self._intance_name = instance_name
         else:
+            print("no preexisting_id")
```

Comment:
```suggestion
```
@umutsoysalansys - this is unacceptable. Make sure you never commit print statements in a PR. It's fine if you want to debug with them locally but you should get used to using the debugger.

Reply:
oh thank you roberto, mea culpa.